setup.py

```diff
@@ -15,8 +15,8 @@
         # catch Debian's custom user site-packages directory.
         lib_paths.append(get_python_lib(prefix="/usr/local"))
     for lib_path in lib_paths:
-        existing_path = os.path.abspath(os.path.join(lib_path, "django"))
-        if os.path.exists(existing_path):
+        existing_path = Path.absolute(Path.joinpath(lib_path, "django"))
+        if Path.exists(existing_path):
```

Comment:
I would expect this to look more like the following:
```suggestion
        path = Path(lib_path) / 'django'
        if path.absolute().exists():
```
Or potentially:
```suggestion
        if Path(lib_path).joinpath('django').absolute().exists():
```
But also note that `pathlib.Path.absolute()` is not ([currently](https://github.com/python/cpython/pull/26153)) documented, public API.